<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddRowDialog</class>
 <widget class="QDialog" name="AddRowDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,3,1">
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="textLabel">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Enter data about your car:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="locationLabel">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Location</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="fuelLabel">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Fuel Type</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="nameLine"/>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="locationLine"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="ownerLabel">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Owner Type</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="transmissionLabel">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Transmission</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="powerLabel">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Power (horse power)</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="capacityLabel">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Engine Capacity</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="seatsLabel">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Seats</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="mileageLabel">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Mileage (km)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="yearLabel">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Year</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="priceLabel">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Price (million ruble)</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QComboBox" name="fuelCB">
       <item>
        <property name="text">
         <string>Petrol</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Diesel</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Propane gas</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Methane gas</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Eco fuel</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Electricity</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Other</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QComboBox" name="transmissionCB">
       <item>
        <property name="text">
         <string>Manual</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Auomatic</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QComboBox" name="ownerCB">
       <item>
        <property name="text">
         <string>First</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Second</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Third</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fourth &amp; Above</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="9" column="2">
      <widget class="QSpinBox" name="seatsSpinB">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QSpinBox" name="yearSpinB">
       <property name="minimum">
        <number>1885</number>
       </property>
       <property name="maximum">
        <number>2024</number>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QSpinBox" name="mileageSpinBox">
       <property name="maximum">
        <number>1000000000</number>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QDoubleSpinBox" name="capacitySpinB">
       <property name="maximum">
        <double>10000000000000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QDoubleSpinBox" name="powerSpinB">
       <property name="maximum">
        <double>1000000000000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="10" column="2">
      <widget class="QDoubleSpinBox" name="priceSpinB">
       <property name="maximum">
        <double>10000000000000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <item alignment="Qt::AlignHCenter">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddRowDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddRowDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
